x-pack/plugins/transform/server/plugin.ts

```diff
@@ -73,10 +53,24 @@ export class TransformServerPlugin implements Plugin<{}, void, any, any> {
       ],
     });
 
-    this.apiRoutes.setup({
-      router,
-      license: this.license,
-      getStartServices,
+    getStartServices().then(([coreStart, { dataViews }]) => {
```

Comment:
do we know if it's safe to register routes in an async function in setup?
Perhaps a safer way to do this is to make `getStatus` in `License` async and have the route wait on getting the license status.

Reply:
It looks like this plugin from the developer examples uses a similar pattern: [examples/search_examples/server/plugin.ts](https://github.com/elastic/kibana/blob/main/examples/search_examples/server/plugin.ts#L56)

Reply:
ok cool. I was just checking because I remember we had race conditions when reregistering features with other plugins asyncly.
That example is very similar, so this must be ok.